.gitlab-ci.yml

```diff
@@ -45,7 +45,7 @@ Manifest-diff:
   script:
     - ./test/cases/diff-manifests.sh
   variables:
-    RUNNER: aws/rhel-9.2-ga-x86_64
+    RUNNER: aws/rhel-9.3-nightly-x86_64
```

Comment:
Let's make this a GA in a follow-up. It doesn't need to be stuff under development.